config-proxy/src/main/java/com/yahoo/vespa/config/proxy/ConfigProxyRpcServer.java

```diff
@@ -9,19 +10,20 @@
 import com.yahoo.vespa.config.protocol.JRTServerConfigRequest;
 import com.yahoo.vespa.config.protocol.JRTServerConfigRequestV3;
 
+import java.io.File;
 import java.lang.*;
-import java.util.Arrays;
-import java.util.Iterator;
-import java.util.Set;
+import java.util.*;
```

Comment:
No * imports

Reply:
Thanks, new laptop, need to configure IDE again.